### PR TITLE
RDO: Remove border from new content if both border and remove new-content username are set

### LINKED ISSUE #742: Reblog Display Options doesn't remove blockquote from new content when both options are checked
If you have the left blockquote and no url/avatars for new content options enabled in RDO, the new content will still have a blockquote, making it look like it's an extension of the previous comment.

### PR DESCRIPTION
**Pull request by dlmarquis**
Fixes #742

### CODE REVIEW

File: Extensions/better_reblogs.js [hunk 1]
```diff
@@ -220,6 +220,16 @@ XKit.extensions.better_reblogs = new Object({
             if (!(this.preferences.margin.value || this.preferences.remove_user_names.value)) {
                 XKit.tools.add_css(".reblog-list-item .reblog-content {margin-left: 3px;}", "better_reblogs");
             }
+
```
Comment (nightpool) [inline Extensions/better_reblogs.js]: Seems to be showing up on my end with tabs instead of spaces. Mind fixing?

Comment (hobinjk) [inline Extensions/better_reblogs.js]: The standard should be tabs. It's odd that this file doesn't conform to what is specified in the [.editorconfig file](https://github.com/new-xkit/XKit/blob/master/.editorconfig).

Comment (nightpool) [inline Extensions/better_reblogs.js]: Pretty sure the project is majority spaces, at least from the parts I've
worked with. Additionally, I've never seen the editorconfig file before.
(And have a very strong dislike of using tabs for code indentation)

Comment (nightpool) [inline Extensions/better_reblogs.js]: Anyway, to kick this up a few meta levels, we should always be consistent
be consistent between files, and changing indentation is out of scope for
the PR

Comment (nightpool) [inline Extensions/better_reblogs.js]: *be consistent WITHIN files

**Comment by dlmarquis** [comment]
@nightpool The indentation should be fixed now.

**Comment by nightpool** [comment]
LGTM @homu r+

**Pull request merged**